src/core_modules/capture-core/components/FiltersForTypes/Date/DateFilter.component.tsx

```diff
@@ -355,20 +347,17 @@ class DateFilterPlain extends Component<Props, State> implements UpdatableFilter
         return (
             <div id="dateFilter">
                 <div>
-                    {/* $FlowFixMe */}
                     <SelectBoxes
-                        optionSet={DateFilter.mainOptionSet}
-                        orientation={orientations.VERTICAL}
+                        optionSet={DateFilterPlain.mainOptionSet}
+                        orientation={orientations.VERTICAL as any}
                         value={value && value.main}
                         onBlur={this.handleMainSelect}
-                        color="secondary"
                         nullable
+                        {...({} as any)}
```

Comment:
Delete `{...({} as any)}` and fix the error by adding `options?: null;` in src/core_modules/capture-core/components/FormFields/Options/SelectBoxes/selectBoxes.types.ts on line 24